### ISSUE TITLE
Cinny 4.0.3 in mobile layout does not show links back to Home

### ISSUE THREAD

**Issue by jman-schief** [ajbura/cinny#1844]
### Describe the bug

When Cinny 4.0.3. switches to the mobile layout, I don't see any link back to the Home. So, if for example I am looking at a chat, I am stuck and cannot switch to another one.

### Reproduction

1. Open Cinny on Firefox
2. Force the mobile layout (either by using the Developer Tools or by reducing the width of the browser window)
3. The "Home" panel/column disappears (correct)
4. I don't see any way to switch to another chat (incorrect)

### Expected behavior

I expect the mobile layout to allow me switching to other chats

### Platform and versions

```shell
1. Linux Debian/trixie (testing channel)
2. Mozilla Firefox 129.0b8 
3. Cinny 4.0.3 (app.cinny.in)
4. Matrix homeserver: (my instance) synapse v1.111.0
```

### Additional context

Cinny <4.x - when using the mobile layout - had a backlink icon on the upper left corner of the window that also showed notifications from other chats

**Comment by ajbura** [comment]
You can use mobile back button to navigate back.

**Comment by jman-schief** [comment]
@ajbura thanks for your comment.

The back button of the browser "walks back" the history of pages previously visited (I think it equals to a `history.back()`). The "back" button in Cinny <4.x took the user to the Home IIRC (I can't verify anymore).

I have no way to go anywhere else if the following content is hidden:
![grafik](https://github.com/user-attachments/assets/66ddcd13-e58d-40e9-b2b7-aff5c946a86b)

**Comment by jman-schief** [comment]
In addition, when I receive a message in another chat I may get a system notification (if I enabled them) but I have absolutely no visual clue in Cinny, which instead I see in the normal web layout, which shows those important visual hints:

![grafik](https://github.com/user-attachments/assets/03f45ec5-1e96-4536-b44c-3510bc8df222)

So, I slightly disagree to classify this as an improvement, feels like a bug or more precisely a regression in the transition to 4.x

(anyway, not pushing for a quick resolution)

**Comment by jman-schief** [comment]
thanks you @ajbura for the fix. I have updated to Cinny 4.1.0 and it feels like a big improvement!
